libraries/abstractions/pkcs11/CMakeLists.txt

```diff
@@ -23,27 +23,78 @@ afr_module_dependencies(
 )
 
 
-# Link to this INTERFACE target to use the default implementation based on mbedtls.
-afr_module(NAME pkcs11_mbedtls INTERFACE)
+if(ECC608_IN_USE)
```

Comment:
will this flag be set to false by default for all non supported boards?

Reply:
yes, not found defaults to false